runbot_gitlab/runbot_repo.py

```diff
@@ -146,7 +147,16 @@ def set_gitlab_ci_conf(token, gitlab_url, runbot_domain, repo_id):
 
 class RunbotRepo(models.Model):
     _inherit = "runbot.repo"
-    uses_gitlab = fields.Boolean('Use Gitlab')
+
+    uses_gitlab = fields.Boolean(string='Use Gitlab')
```

Comment:
That's very interesting hbrunn

The initial thought was that "explicit is always better than implicit"
So in conclusion for fields that have the exact same name as the variable omit the string parameter (both positional and keyword argument) ?